### ISSUE TITLE
Add support to release aarch64 wheels

### ISSUE THREAD

**Issue by odidev** [duckdb/duckdb#1677]
Problem
---------
On aarch64, pip install duckdb builds the wheels from source code and then install it. It requires user to have development environment installed on his system. also, it take some time to build the wheels than downloading and extracting the wheels from pypi.

Resolution
-----------
On aarch64, pip install duckdb should download the wheels from pypi

@hannesmuehleisen, Please let me know your interest on releasing aarch64 wheels. I can help in this.

**Comment by hannesmuehleisen** [comment]
Yes we are very much interested in this. Just approved your PR to run tests.

**Comment by hannes** [comment]
This is supported now
